dbms/src/Interpreters/Context.cpp

```diff
@@ -1222,8 +1222,8 @@ void Context::setCurrentQueryId(const String & query_id)
             } words;
         } random;
 
-        random.words.a = thread_local_rng();
-        random.words.b = thread_local_rng();
+        random.words.a = thread_local_rng(); //-V656
```

Comment:
Any description?

Reply:
Suppresstion. It's required by PVS-Studio to be in exactly this format.
The only way is to write a comment in a line above, but it's too cumbersome to do it for every suppression.